digest.gemspec

```diff
@@ -14,12 +14,12 @@ Gem::Specification.new do |spec|
 
   spec.files         = [
     "LICENSE.txt", "README.md",
-    "ext/digest/bubblebabble/bubblebabble.c", "ext/digest/bubblebabble/extconf.rb", "ext/digest/defs.h",
+    "ext/digest/bubblebabble/bubblebabble.c", "ext/digest/bubblebabble/extconf.rb", "ext/digest/bubblebabble/lib/bubblebabble.rb", "ext/digest/defs.h", 
     "ext/digest/digest.c", "ext/digest/digest.h", "ext/digest/digest_conf.rb", "ext/digest/extconf.rb",
     "ext/digest/md5/extconf.rb", "ext/digest/md5/md5.c", "ext/digest/md5/md5.h", "ext/digest/md5/md5cc.h",
-    "ext/digest/md5/md5init.c", "ext/digest/rmd160/extconf.rb", "ext/digest/rmd160/rmd160.c",
-    "ext/digest/rmd160/rmd160.h", "ext/digest/rmd160/rmd160init.c", "ext/digest/sha1/extconf.rb",
-    "ext/digest/sha1/sha1.c", "ext/digest/sha1/sha1.h", "ext/digest/sha1/sha1cc.h",
+    "ext/digest/md5/md5init.c", "ext/digest/md5/lib/md5.rb", "ext/digest/rmd160/extconf.rb", "ext/digest/rmd160/rmd160.c",
+    "ext/digest/rmd160/rmd160.h", "ext/digest/rmd160/rmd160init.c", "ext/digest/rmd160/lib/rmd160.rb", "ext/digest/sha1/extconf.rb",
+    "ext/digest/sha1/sha1.c", "ext/digest/sha1/sha1.h", "ext/digest/sha1/sha1cc.h", "ext/digest/sha1/lib/sha1.rb",
```

Comment:
These added files should be inside `if RUBY_ENGINE == 'jruby'` block.

Reply:
java sources don't have to be part of a released gem, so I removed them instead